modules/tensor_mechanics/src/materials/StressUpdateBase.C

```diff
@@ -0,0 +1,42 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "StressUpdateBase.h"
+
+#include "MooseMesh.h"
+
+template<>
+InputParameters validParams<StressUpdateBase>()
+{
+  InputParameters params = validParams<Material>();
+  params.addClassDescription("Calculates the effective inelastic strain increment required to return the isotropic stress state to a J2 yield surface.  This class is intended to be a parent class for classes with specific constitutive models.");
+  params.addParam<std::string>("base_name", "Optional parameter that allows the user to define multiple mechanics material systems on the same block, i.e. for multiple phases");
+
+  params.addParam<bool>("compute", false, "The return stress increment classes are intended to be iterative materials");
```

Comment:
Well, if the parameter is already added in `Material` then set+suppress would be the right thing here.